src/modal.js

```diff
@@ -283,7 +283,7 @@ ddm.modal = (function ($) {
       }
     });
 
-    $element.on('wheel', function () {
+    $element.on('wheel', function (event) {
       event.preventDefault();
     });
```

Comment:
Actually, looking at the changes made to the modal recently, there's already a [handler for wheel above](https://github.com/deseretdigital-ui/ddm-modal/pull/7/files#diff-d44de49178456a8a04cc5899966b26e9L127), so I'm not even sure why this is here? I'll try taking it out and see what happens.